### DISCUSSION TITLE
github is'n working in my visual studio code

### DISCUSSION THREAD

**Discussion by arif1408** [community/community#11214] [Copilot Conversations]
github copilot is'n working in my vs code

**Comment by davecheney** [comment]
Hello. I’m sorry you’re having difficulties. Could you tell me what you did, what you expected, and what happened instead?

**Comment by arif1408** [comment]
My problem was done, that issue cause my vs code want to update to newer.
Version admin

Pada tanggal Rab, 9 Feb 2022 11.33, Dave Cheney ***@***.***>
menulis:

**Reply by davecheney** [reply]
Yup, copilot and VSCode are written in software, so the first step in solving any problem is to update.
